indigo_app/views/tasks.py

```diff
@@ -100,9 +100,29 @@ def get_context_data(self, **kwargs):
         # merge actions and comments
         actions = task.action_object_actions.all()
         task_content_type = ContentType.objects.get_for_model(self.model)
-        comments = Comment.objects\
+        comments = list(Comment.objects\
             .filter(content_type=task_content_type, object_pk=task.id)\
-            .select_related('user')
+            .select_related('user'))
+        
+        # get the annotation for the particular task
+        task_annotation = Annotation.objects.filter(task=task).first()
+
+        # for the annotation that is linked to the task, get all the replies
+        if task_annotation:
+            fake_comments = []
+            # get the replies to the annotation
+            annotation_replies = Annotation.objects.filter(in_reply_to=task_annotation)
```

Comment:
You also need the user object, so be sure to use `select_related('user')` otherwise it will go to the database each time you access the user on the reply object.